modules/standard/CTypes.chpl

```diff
@@ -1224,12 +1222,10 @@ module CTypes {
 
     For example:
 
-    .. code-block:: chapel
-
-      var myString = "Hello!";
-      on differentLocale {
-        writef("%s", myString.localize().c_str());
-      }
+    .. literalinclude:: ../../../../test/library/standard/CTypes/doc-examples/CptrExamples.chpl
```

Comment:
I think you mean `CstrExamples`?  Looks like you put `CptrExamples` here and in the one below

Reply:
Good catch, thanks!